README.md

```diff
@@ -1,8 +1,77 @@
-# Jarivs Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
 
+# Xinyu Wang . Jarvis Consulting
+Driven and analytical professional with a bachelor's level education in electrical engineering from Carleton University.
+Apply engineering expertise to projects on devices and product development, programming and software design, and electronic manufacturing. 
+Collaborate on teams to manage multifaceted projects and synthesize complex concepts into engineering solutions. 
+Demonstrate proven ability to apply engineering skills to a wide range of electrical and software engineering projects. 
+
+## Technical Acumen
+
+**Proficient:** Java, Bash, SQL, Agile/Scrum, C,
+
+**Competent:** MATLAB, Python, Embedded System, C++, Junit, Lambda/Stream API, Mockito, Spring Framework
+
+**Familiar:** Google Cloud Platform, Git, Tensorflow
+
+**Hardware Design Tool:**  Cadence, ADS, HFSS, Simulink
+
+## Development Projects in Jarvis
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_shawn](https://github.com/jarviscanada/jarvis_data_eng_shawn)
+
+- **[Cluster Monitor](./linux_sql)**:   
+  * Developed a cluster monitoring bash agent that records hardware specifications and monitors
+resource usage (per minute) of all nodes in a cluster. Provisioned PostgreSQL database in nodes to store
+information of all nodes with the help of the bash agent installed in all of them. Utilized cron job to schedule the
+collection of resource usage information every minute. Coded SQL queries to generate reports for some business questions.
+                                      
+- **[Core Java Apps](./core_java)**:  
+  * Designed a Java8 based application called JavaGrep that searches all files in a directory
```

Comment:
Designed and programmed ~~called JavaGrep~~ 

Please mention that you implemented it with Java Lamdba and Stream APIs